import sys
def on_exception(e):
    print("Error Occured, Bye...")
    sys.exit(0)